free-courses-en.md

```diff
@@ -187,7 +188,10 @@
 * [Docker Tutorial for Beginners](https://www.youtube.com/watch?v=fqMOX6JJhGo) - FreeCodeCamp
 * [Learn Docker, Container Runtimes, Builders and Registries](https://www.katacoda.com/courses/container-runtimes) - Katacoda
 
-
+### flutter
+* [flutter Tutorial](https://www.udacity.com/course/build-native-mobile-apps-with-flutter--ud905) - Udacity
+* [flutter Tutorial for Beginners](https://www.udemy.com/course/learn-flutter-beginners-course/) - Udemy
+* [flutter Crash Course](https://fluttercrashcourse.com/courses/basics/lessons/materialapp-scaffold-appbar-text) - Flutter CrashCourse
```

Comment:
Flutter should have a capital F.
There should also be line breaks between sections.

Reply:
Sure i will do the changes